### ISSUE TITLE
Autocomplete for table names

### ISSUE THREAD

**Issue by gramster** [GoogleCloudPlatform/datalab#646]
Was thinking about this and wondered - could we take advantage of the normal autocomplete mechanism to provide table autocompletion if the user entered '[' followed by TAB key?

**Comment by gramster** [comment]
I did some experimentation and it didn't look to promising, in that the autocompleter didn't seem to get invoked; maybe it requires a '.' before TAB. We could fix this as we control the FE but it is not as simple as I hoped.

**Comment by nikhilk** [comment]
The autocompletion keystroke is Shift+Tab IIRC.

**Comment by gramster** [comment]
Yes, I'm just being lazy in my description; still doesn't seem to work. Trying to use this hook:

    self.set_hook('complete_command', table_completer, re_key = '[Ff][Rr][Oo][Mm][ \t]+\[')
